missioncontrol/v0/accesses.py

```diff
@@ -444,11 +446,14 @@ def get_access(access_id):
     access = Access.from_id(access_id, base_url=base_url)
     return access.to_dict()
 
-def get_track(access_id):
+def get_track(access_id, **kwargs):
```

Comment:
I definitely prefer being explicit here, especially given you have "acceptable kwargs" filtering below.
Am I missing something?

Reply:
Didn't know about schema defaults.
Otherwise either meant I needed to handle a potential None making it down, or removing the kwargs (eg user) that aren't accepted